package.json

```diff
@@ -31,8 +33,10 @@
     "danger": "^3.1.8",
     "eslint": "^4.18.2",
     "eslint-config-sexy": "^3.2.1",
+    "express-bunyan-logger": "^1.3.2",
```

Comment:
Mm, it seems this belongs in devDependencies? Same with nodemon if you wanna use it in production

Reply:
can be moved to deps tho